applicationinsights.json

```diff
@@ -0,0 +1,12 @@
+{
+  "role": {
+    "name": "offender-case-notes"
```

Comment:
Is this the correct name for the application insights? (I have no idea, but offender-case-notes isn't what I'd expect.)

Reply:
Thanks for catching that, it's a copy-and-paste error!